plugins/modules/cloud/misc/proxmox.py

```diff
@@ -140,9 +140,10 @@
     type: str
   unprivileged:
     description:
-      - Indicate if the container should be unprivileged
+      - Indicate if the container should be unprivileged.
+      - The default value for this param is C(no) but that is being deprecated.
+      - It will be replaced with C(yes) in community.general 7.0.0
```

Comment:
This will also become part of the official, published documentation, so please use full words and punctuation:
```suggestion
      - The default value for this parameter is C(no) but that is being deprecated.
      - It will be replaced with C(yes) in community.general 7.0.0.
```